pype/hosts/nukestudio/tags.py

```diff
@@ -71,8 +71,8 @@ def add_tags_from_presets():
     # Get project task types.
     tasks = io.find_one({"type": "project"})["config"]["tasks"]
     nks_pres_tags["[Tasks]"] = {}
-    for task in tasks:
-        nks_pres_tags["[Tasks]"][task["name"]] = {
+    for task_name, _ in tasks.items():
```

Comment:
I think variable `task_name` should be `task_type` and to avoid the underscore variable you can just use `for task_name in tasks.keys():`

Reply:
Modified, makes sense.